pkgs/top-level/all-packages.nix

```diff
@@ -25062,6 +25062,8 @@ with pkgs;
 
   unixODBCDrivers = recurseIntoAttrs (callPackages ../development/libraries/unixODBCDrivers { });
 
+  unl0kr = callPackage ../tools/misc/unl0kr { };
```

Comment:
Nowadays you can use `pkgs/by-name` structure. Just throw your default.nix file inside `pkgs/by-name/un/unl0kr/package.nix` and

```suggestion
```